### PR TITLE
Legal updates: Update Location, Update Relationship Section, Update References to 1Social, and Change One to 1.

### PR DESCRIPTION
**Pull request by wgyt**

### CODE REVIEW

**Comment by wgyt** [comment]
todo:

- [ ] update "Your relationship with OneDot"
- [ ] "Using OneDot products" needs the reference to 1social removed and the part "Using OneDot services on behalf of an organization"
- [ ] Change OneDot to 1Dot

**Comment by wgyt** [comment]
@RedGuy12
> your organization’s administrator may assign a 1Auth Account to you.
          That administrator might require you to follow additional rules and
          may be able to access or disable your 1Auth Account.

1Auth Account's don't exist, yet, can https://github.com/onedotprojects/auth/issues/15 be prioritized?

**Comment by RedGuy12** [comment]
i want to use `one` in all of our products instead of `1` pls

it just sounds better


I'm just wailing for https://discord.com/channels/825806006554329109/854033008349020160/854411740375679026

**Comment by wgyt** [comment]
> i want to use `one` in all of our products instead of `1` pls
> 
> it just sounds better
> 
> I'm just waiting for https://discord.com/channels/825806006554329109/854033008349020160/854411740375679026

***the website, the logos, and most other stuff uses 1, consistency is important***

**Comment by RedGuy12** [comment]
> **_the website, the logos, and most other stuff uses 1, consistency is important_**

**_we can change them_**

- "one" sounds better
- "one" looks better
- for all our accounts (gh, replit, twitter, npm, etc) we use "OneDot Projects" or "onedotprojects"
- the domain is literally **1**dot

**Comment by wgyt** [comment]
> > **_the website, the logos, and most other stuff uses 1, consistency is important_**
> 
> **_we can change them_**
> 
> * "one" sounds better
> * "one" looks better
> * for all our accounts (gh, replit, twitter, npm, etc) we use "OneDot Projects" or "onedotprojects"
> * the domain is literally **one**dot.cf

true, but we can also change all of our accounts to "1Dot" or "1DotProjects", and get a new domain.

**Comment by RedGuy12** [comment]
> > > **_the website, the logos, and most other stuff uses 1, consistency is important_**
> > 
> > 
> > **_we can change them_**
> > 
> > * "one" sounds better
> > * "one" looks better
> > * for all our accounts (gh, replit, twitter, npm, etc) we use "OneDot Projects" or "onedotprojects"
> > * the domain is literally **one**dot.cf
> 
> true, but we can also change all of our accounts to "1Dot" or "1DotProjects", and get a new domain.

noooo

get on discord

**Comment by RedGuy12** [comment]
> **_the website, the logos, and most other stuff uses 1, consistency is important_**

the only things that use 1 are:
- logos
- transifex
- anytime it's mentioned in text

everything else uses one

changing tags/usernames could cause confusion and complications

we already have logos that use "one", we just need the padding to be removed and new favicons (which i am doing now)

**Comment by wgyt** [comment]
> > **_the website, the logos, and most other stuff uses 1, consistency is important_**
> 
> the only things that use 1 are:
> 
> * logos
> * transifex
> * anytime it's mentioned in text
> 
> everything else uses one
> 
> changing tags/usernames could cause confusion and complications
> 
> we already have logos that use "one", we just need the padding to be removed and new favicons (which i am doing now)

we can also keep the old username by transferring all of the repos/repls to the new team and setting redirects up

we also have logos without padding that have "one": https://onedot.cf/onedotbrandv3/<ProductName>.<svg or png>

**Comment by retronbv** [comment]
> > **_the website, the logos, and most other stuff uses 1, consistency is important_**
> 
> 
> 
> **_we can change them_**
> 
> 
> 
> - "one" sounds better
> 
> - "one" looks better
> 
> - for all our accounts (gh, replit, twitter, npm, etc) we use "OneDot Projects" or "onedotprojects"
> 
> - the domain is literally **one**dot.cf
> 
> 
> 
> 

How does one sound better if it literally sounds the same as 1

**Comment by wgyt** [comment]
> How does one sound better if it literally sounds the same as 1

**Comment by RedGuy12** [comment]
ok it **looks** better

**Comment by RedGuy12** [comment]
how about we leave 1/one changes out of this pr

we can add them back to #3

**Comment by RedGuy12** [comment]
> @RedGuy12
> 
> > your organization’s administrator may assign a 1Auth Account to you.
> > That administrator might require you to follow additional rules and
> > may be able to access or disable your 1Auth Account.
> 
> 1Auth Account's don't exist, yet, can [onedotprojects/auth#15](https://github.com/onedotprojects/auth/issues/15) be prioritized?

onedotprojects/auth#15 is a huge project

not gonna happen quickly

**Review by RedGuy12 — APPROVED** [review]
tbh i dont really care what happens to this lol


just keep the `one`

File: pages/index.vue [hunk 1]
```diff
@@ -68,10 +68,10 @@
         <img
           src="/onedotbrandv3/color/1Api.svg"
           class="object-fill w-full h-full rounded-lg rounded-b-none"
-          alt="1API"
+          alt="1Api"
         />
         <div class="prose">
-          <h2>1API will be a package for 1Host that makes making apis easy</h2>
+          <h2>1Api will be a package for 1host that makes making apis easy</h2>
```
Comment (wgyt) [inline pages/index.vue]: god, the capitalization issues

File: pages/index.vue [hunk 2]
```diff
@@ -205,7 +205,7 @@
           alt="1CSS"
         />
         <div class="prose">
-          <h2>1CSS is a css framework</h2>
+          <h2>1css is a css framework</h2>
```
Comment (wgyt) [inline pages/index.vue]: god, the capitalization issue

File: pages/index.vue [hunk 3]
```diff
@@ -228,12 +228,12 @@
         <img
           src="/onedotbrandv3/color/1host.svg"
           class="object-fill w-full h-full rounded-lg rounded-b-none"
-          alt="1Host"
+          alt="1host"
```
Comment (wgyt) [inline pages/index.vue]: god, the capitalization issues

File: pages/legal.vue [hunk 4]
```diff
@@ -16,175 +16,172 @@
             <img height="54px" src="/onedotbrandv3/color/1dot.svg" />
           </a>
         </div>
-        <div class="w-full md:w-auto md:flex-grow md:flex md:items-center">
+            <div class="w-full md:w-auto md:flex-grow md:flex md:items-center">
```
Comment (wgyt) [inline pages/legal.vue]: wtf

File: pages/legal.vue [hunk 5]
```diff
@@ -16,175 +16,172 @@
...
               -mx-4
               pt-4
               border-t
-              md:flex-row
-              md:items-center
-              md:mx-0
-              md:mt-0
-              md:pt-0
-              md:mr-4
+              md:flex-row md:items-center md:mx-0 md:mt-0 md:pt-0 md:mr-4
```
Comment (wgyt) [inline pages/legal.vue]: wtf

File: pages/legal.vue [hunk 6]
```diff
@@ -16,175 +16,172 @@
...
               lg:mr-8
               md:border-0
             "
           >
             <li>
               <a
                 class="block px-4 py-1 md:p-2 lg:px-4"
-                href="https://github.com/1Dotprojects"
+                href="https://github.com/onedotprojects"
```
Comment (wgyt) [inline pages/legal.vue]: oops

File: pages/legal.vue [hunk 7]
```diff
@@ -16,175 +16,172 @@
...
                 >Github</a
               >
             </li>
             <li>
-              <a class="block px-4 py-1 md:p-2 lg:px-4" href="/legal">Legal</a>
+              <a
+                class="block px-4 py-1 md:p-2 lg:px-4"
+                href="/legal"
+                >Legal</a
+              >
```
Comment (wgyt) [inline pages/legal.vue]: wtf

File: pages/legal.vue [hunk 8]
```diff
@@ -16,175 +16,172 @@
...
-        (-william27- / wgyt), Graham S H (GrahamSH / GrahamSH-LLK), Paul Reid
-        (Redguy12 / Redguy13), Retronbv, thecrewmate876 (thecoder876), Touchcreator,
-        and Xan.
-      </p>
-      <h4 id="what-you-can-expect-from-us">What you can expect from us</h4>
-      <ul>
+        terms. When we speak of “OneDot,” “we,” “us,” and “our,” we mean OneDot
+        Projects, a group of developers including William Horning
+        (-william27-/wgyt), Touchcreator, BakeAPotato, -Xanimation-, retronbv,
+        RedGuy (Paul Reid), and others.
```
Comment (wgyt) [inline pages/legal.vue]: Don't revert this

File: pages/legal.vue [hunk 9]
```diff
@@ -16,175 +16,172 @@
...
-      </h4>
-      <p>
+        </ul>
+        <h3 id="using-onedot-products">Using OneDot products</h3>
+        <h4 id="your-account">Your Account</h4>
+        <p>
+        If you meet the age requirements you can create a OneAuth Account for your
+        convenience. Some services require that you have a OneAuth Account to work
+        — for example, to use OneSocial, you need a OneAuth Account so that you
+        can comment and chat.
```
Comment (wgyt) [inline pages/legal.vue]: bruh dont revert this

**Review by wgyt — COMMENTED** [review]
three words: what the fuck

**Comment by wgyt** [comment]
> > @RedGuy12
> > > your organization’s administrator may assign a 1Auth Account to you.
> > > That administrator might require you to follow additional rules and
> > > may be able to access or disable your 1Auth Account.
> > 
> > 
> > 1Auth Account's don't exist, yet, can [onedotprojects/auth#15](https://github.com/onedotprojects/auth/issues/15) be prioritized?
> 
> [onedotprojects/auth#15](https://github.com/onedotprojects/auth/issues/15) is a huge project
> 
> not gonna happen quickly

ok lol

File: pages/legal.vue (hunk 8 above)
Comment (RedGuy12) [reply pages/legal.vue]: i didn't remove it i moved it

**Pull request merged**